### DISCUSSION TITLE
Annotation Tool Korean Character Support

### DISCUSSION THREAD

**Discussion by kohyuk91** [AcademySoftwareFoundation/OpenRV#495] [Q&A] [answered]
Hi everyone,

Is it not possible to write Korean characters with the Annotation tool?
Seems like it only supports English characters.

Thanks

I'm running OpenRV on Rocky Linux 8.9

**Comment by bernie-laberge** [comment, marked as answer]
Hello @kohyuk91,
Unfortunately it is not currently possible to write non English characters with the Annotation tool.

We are trying to work on it whenever we get the chance as you can see by these PRs:
https://github.com/AcademySoftwareFoundation/OpenRV/pull/524
https://github.com/AcademySoftwareFoundation/OpenRV/pull/524

I apologize for the very late reply!
